package.json

```diff
@@ -31,7 +31,7 @@
     "estree-walker": "2.0.2",
     "glob": "^10.4.5",
     "graceful-fs": "^4.2.9",
-    "node-gyp-build": "^4.2.2",
+    "node-gyp-build": "~4.7.1",
```

Comment:
`^4.2.2` already covers all of 4.x because of the upcaret

Why is this being changed to a tilde?

Reply:
Could be that the 4.8 did break stuff - will have a second look

Reply:
@styfle - reverted